pages/advice_video_page.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'page'
+
+class AdviceVideoPage < Page
+
+  def marker
+    page.find(:xpath, "//h1[@class = 'display-1']", visible: false).text
```

Comment:
You don't need `xpath` here, use `css` locator for simple searches, like this one.